test/statsCases/warnings-uglifyjs/webpack.config.js

```diff
@@ -1,10 +1,12 @@
+//eslint-disable-next-line
```

Comment:
I don't see any reason to disable eslint here. Can't the problem be fixed with code or configuration?